### DISCUSSION TITLE
svg-sprite

### DISCUSSION THREAD

**Discussion by CalledByThe4ire** [cvgellhorn/webpack-boilerplate#370] [Ideas]
Hey there! Could you please add support for svg symbols?

**Comment by cvgellhorn** [comment]
Hey Andrey,

sorry for my late response.

I’m pretty busy at the moment and sadly don’t have time to look into SVG support, but I’m happy to accept ideas via pull request if you have anything specific in mind?

Cheers
Chris

**Reply by CalledByThe4ire** [reply]
Smth like this:
```
const SpriteLoaderPlugin = require('svg-sprite-loader/plugin');

plugins: [
    new SpriteLoaderPlugin(),
  ],
module: {
    rules: [
      {
        test: /\.svg$/,
        use: [{ loader: 'svg-sprite-loader', options: {} }],
      },
    ],
  },

**Reply by cvgellhorn** [reply]
Thanks for the suggestion, I will have a look into that plugin and it's usage.

Cheers Chris

**Comment by cvgellhorn** [comment]
Hey @CalledByThe4ire,

SVG's are now supported. 

I decided to go with the webpack recommendation and use a combination of the raw-loader and imagemin-svgo plugin. I didn’t like the way you have to insert the SVG into the DOM and wanted an easier way. If I need more flexibility later on, I might reconsider the svg-sprite-loader.


Cheers Chris

**Reply by CalledByThe4ire** [reply]
Thanks a lot, Chris. Awesome!
